runtime-modules/content-directory/src/permissions.rs

```diff
@@ -107,8 +107,8 @@ pub fn perform_curator_in_group_auth<T: Trait>(
     // Ensure curator authorization performed succesfully
     ensure_curator_auth_success::<T>(curator_id, account_id)?;
 
-    // Ensure CuratorGroup under given curator_group_id exists, retrieve corresponding one
-    let curator_group = Module::<T>::ensure_curator_group_exists(curator_group_id)?;
+    // Retrieve corresponding curator group
+    let curator_group = Module::<T>::curator_group_by_id(curator_group_id);
```

Comment:
Why was this check removed? 

This would return a default constructed group if the `curator_group_id` was not valid, and I assume that the next ensure check would trigger `ERROR_CURATOR_GROUP_IS_NOT_ACTIVE`, which would stop the call. This is however not safe. It will depend on behaviour of `is_active` in default mode in a way which is not clear going forward, and it also returns a misleading dispatch error.